src/net/xds/client.rs

```diff
@@ -243,34 +244,32 @@ impl MdsClient {
         let handle = tokio::task::spawn(
             async move {
                 tracing::trace!("starting relay client delta stream task");
-                let mode = self.mode.clone();
-                let agent = self.mode.unwrap_agent();
-                let interval = mode.idle_request_interval();
+                let interval = rt_config.idle_request_interval;
 
                 loop {
                     {
                         let control_plane =
-                            super::server::ControlPlane::from_arc(config.clone(), mode.clone());
+                            super::server::ControlPlane::from_arc(config.clone(), interval);
                         let mut stream = control_plane.delta_aggregated_resources(stream).await?;
-                        agent.relay_is_healthy.store(true, Ordering::SeqCst);
+                        rt_config.relay_is_healthy.store(true, Ordering::SeqCst);
 
                         loop {
                             let timeout = tokio::time::timeout(interval, stream.next());
 
                             match timeout.await {
                                 Ok(Some(result)) => {
                                     let response = result?;
-                                    tracing::debug!("received delta discovery response");
+                                    tracing::trace!("received delta discovery response");
                                     ds.send_response(response).await?;
                                 }
                                 _ => break,
                             }
                         }
                     }
 
-                    agent.relay_is_healthy.store(false, Ordering::SeqCst);
+                    rt_config.relay_is_healthy.store(false, Ordering::SeqCst);
 
-                    tracing::warn!("lost connection to relay server, retrying");
+                    //tracing::warn!("lost connection to relay server, retrying");
```

Comment:
Why is this commented out?

Reply:
I was/am debugging some tests and there were a few extremely spammy trace lines, will add them back.